site/data/dynamo.ts

```diff
@@ -67,6 +66,29 @@ export const getPublishedDisruptionsDataFromDynamo = async (id: string): Promise
     return disruptionIds?.map((id) => collectDisruptionsData(dbData.Items || [], id)).filter(notEmpty) ?? [];
 };
 
+export const getSubmittedDisruptionsDataFromDynamo = async (id: string): Promise<Disruption[]> => {
+    logger.info("Getting disruptions data from DynamoDB table...");
```

Comment:
Should we change the log here and the one in getPublishedDisruptionsDataFromDynamo so that they're different? As they're doing different things.

Reply:
done